docs/fetchJson.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: default
+title: "fetchJson"
+---
+
+# `fetchJson`
+
+React-admin includes a `fetchJson` utility function to make HTTP calls. It's a wrapper around the browser's `fetch` function, which adds the following features:
+
+-   It adds the `Content-Type='application/json'` header to all non GET requests
+-   It adds the `Authorization` header with optional parameters
+-   It makes it easier to add custom headers to all requests
+-   It handles the JSON decoding of the response
+-   It handles HTTP errors codes by throwing an `HttpError`
+
+## Usage
+
+You can use it to make HTTP calls directly, to build a custom [`dataProvider`](./DataProviderIntroduction.md), or pass it directly to any `dataProvider` that supports it, such as [`ra-data-simple-rest`](https://github.com/marmelab/react-admin/tree/master/packages/ra-data-simple-rest).
+
+```jsx
+import { fetchUtils, Admin, Resource } from 'react-admin';
+import simpleRestProvider from 'ra-data-simple-rest';
+
+const httpClient = async (url, options = {}) => {
+    const { status, headers, body, json } = fetchUtils.fetchJson(url, options);
+    console.log('fetchJson result', { status, headers, body, json });
+    return { status, headers, body, json };
+}
+const dataProvider = simpleRestProvider('http://path.to.my.api/', httpClient);
+
+const App = () => (
+    <Admin dataProvider={dataProvider}>
+        <Resource name="posts" list={PostList} />
+    </Admin>
+);
+```
+
+**Tip:** `fetchJson` is included in the `fetchUtils` member of the `react-admin` package.
+
+### Parameters
```

Comment:
We don't need 2 levels of heading here. Make it a top-level heading (`###` => `##`)